.github/workflows/ci.yaml

```diff
@@ -0,0 +1,73 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - release/v*
+    paths-ignore:
+      - '.gitignore'
+      - 'CODEOWNERS'
+      - 'LICENSE'
+  pull_request:
+    paths-ignore:
+    - '.gitignore'
+    - 'CODEOWNERS'
+    - 'LICENSE'
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    name: CI
+    strategy:
+      matrix:
+        archBox:
+        - { arch: amd64, vmArch: x64 }
+        - { arch: arm64, vmArch: arm64 }
+    runs-on: runs-on,runner=1cpu-linux-${{ matrix.archBox.vmArch }},run-id=${{ github.run_id }}
+    steps:
+      - name : Checkout repository
+        # https://github.com/actions/checkout/releases/tag/v4.1.1
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Setup Go
+        uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
+        # https://github.com/actions/setup-go/releases/tag/v5.0.0
+        with:
+          go-version-file: 'go.mod'
+
+      - name: install K3d
+        run: ./.github/workflows/scripts/install-k3d.sh
+        env:
+          K3D_VERSION: latest
+
+      - name: ci
+        run: make ci
+
+      - name: setup cluster
+        run: ./.github/workflows/scripts/setup-cluster.sh
+        env:
+          CLUSTER_NAME: webhook
+          K3S_VERSION: v1.28.11-k3s1
+          ARCH: "${{ matrix.archBox.arch }}"
+
+      - name: import image
+        run: k3d image import dist/rancher-webhook-image.tar -c webhook
+
+      - name: start rancher
+        run: ./.github/workflows/scripts/start-rancher.sh
+        env:
+          VERSION: "2.8"
```

Comment:
Imo this should use the same mechanism that we use for release/v0.5 because right now this only allows running tests on a previously released version of Rancher.

eg: If I add a `Foo` object in rancher 2.8, the CI will not pick it up because it will install a released version of rancher. We should use the v2.8-head image instead and its chart so we can test unreleased changes to CRDs.

Reply:
done